src/7ABB10.c

```diff
@@ -19,8 +19,10 @@ void func_802D2604(GObj* obj) {
     animal* animal = obj->data.animal;
 
     runGObjProcess(obj, spawnStaryuAtGeo, 1, 1);
-    
+
+    // clang-format off
```

Comment:
needing these is un-ideal. can we just not enforce newline rules? Or maybe a leading slash + newline will work here